lib/config.js

```diff
@@ -285,6 +291,18 @@ Config.prototype.update = function (options) {
     this.channel = options.channel
   }
 
+  if (this.officialBuild && process.platform !== 'linux') {
```

Comment:
can we check `buildConfig == 'Release'` instead of `officialBuild`?

Reply:
Yep, only enable with Release build seems fine.
Done.